apps/emqx_authn/src/emqx_authn_app.erl

```diff
@@ -23,7 +23,9 @@
 %% Application callbacks
 -export([
     start/2,
-    stop/1
+    stop/1,
+    chain_configs/0,
```

Comment:
Do we need to export these funs?

Reply:
sorry for this, I just export for debug. removed